lms/templates/support/contact_us.html

```diff
@@ -11,7 +13,7 @@
 
 <%block name="title">
     <title>
-        ${_("Contact US")}
+        ${_("{platform_name} Support: Contact Us".format(platform_name=platform_name))}
```

Comment:
The nesting looks a bit odd in this. http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html#do-not-append-strings-or-interpolate-values has some examples of how this should be done.